todo_app/Scripts/activate

```diff
@@ -0,0 +1,66 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
All of your venv files should be added to your `gitignore`. Only the `requirements.txt` are necessary for someone to run your project.

Reply:
As well as all your site-packages scripts, like `todo_app/Scripts/pip.exe`, etc.